SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.h

```diff
@@ -22,3 +23,5 @@ void GenerateSubobjectSchema(FComponentIdGenerator& IdGenerator, UClass* Class,
 
 // Generates schema for RPC endpoints.
 void GenerateRPCEndpointsSchema(FString SchemaPath);
+
+void AddComponentId(uint32 ComponentId, uint32 (&SchemaComponents)[ESchemaComponentType::SCHEMA_Count], ESchemaComponentType ComponentType);
```

Comment:
Would it be possible to use our defined types here. Worker_ComponentId. If we can, use it throughout this PR.
Also a major nit but something I think we are trying to do in the gdk: value args that shouldn't change should be const.

Reply:
changed to using worker_componentid to wherever possible while it still builds- the reason I used uint32 is because it was used elsewhere here, do you think it is worth a ticket to go through the codebase to find any usages of uint32 that currently exist that actually mean a component ID?

Also made read only arguments const (I think) throughout

Reply:
We could add a ticket. I don't think it would ever get done. It's a tricky one :/

Reply:
fair enough, I'll leave it then